Proiect1/Proiect1/CircusModel/Circus.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Proiect1.Animals;
+using Proiect1.Interface;
+using Proiect1.Presentation;
+using System.Collections.Generic;
+
+namespace Proiect1.CircusModel
+{
+    internal class Circus
+    {
+        private readonly Arena arena;
+        private readonly List<IAnimal> animals;
+        private readonly string CircusName = "Circus Maximus";
+
+        public Circus(Arena arena)
+        {
+            this.arena = arena;
+            animals = new List<IAnimal>();
+            animals.Add(new Elephant("Babar"));
+            animals.Add(new Lion("King"));
+            animals.Add(new Snake("Serphent"));
+            animals.Add(new Wolf("Alpha"));
+            animals.Add(new Monkey("Wukong"));
```

Comment:
LoL reference; still, nobody asked for the name

Reply:
no more names just sounds and types (species)